lms/djangoapps/courseware/date_summary.py

```diff
@@ -204,6 +205,9 @@ class VerifiedUpgradeDeadlineDate(DateSummary):
 
     @property
     def link(self):
+        ecommerce_service = EcommerceService()
+        if ecommerce_service.is_enabled(self.user):
+            return ecommerce_service.payment_page_url()
         return reverse('verify_student_upgrade_and_verify', args=(self.course.id,))
```

Comment:
That part of the code should be investigated but it isn't part of this ticket so I've created a new ticket for it: https://openedx.atlassian.net/browse/SOL-1714